src/core/etl/src/Flow/ETL/Pipeline/Execution/ExecutionPlan.php

```diff
@@ -8,7 +8,7 @@
 use Flow\ETL\Pipeline\Execution\Plan\FilesystemOperations;
 use Flow\ETL\Pipeline\Pipes;
 
-final class LogicalPlan
+final class ExecutionPlan
```

Comment:
Technically speaking, looking at the current implementation, it's more a Physical than a Logical plan, but for now, lets call it an Execution Plan which IMO it's a most self-descriptive name for what this class is really responsible for.